tests/solids/diamondC_1x1x1-Gaussian_pp/CMakeLists.txt

```diff
@@ -125,11 +125,11 @@ qmc_run_and_check(
 if(QMC_MIXED_PRECISION)
   list(APPEND DET_DIAMOND_VMCBATCH_SCALARS "totenergy" "-10.82674512 0.000222")
   list(APPEND DET_DIAMOND_VMCBATCH_SCALARS "kinetic" "12.23702380 0.000221")
-  list(APPEND DET_DIAMOND_VMCBATCH_SCALARS "potential" "-23.06376891 0.000005")
+  list(APPEND DET_DIAMOND_VMCBATCH_SCALARS "potential" "-23.06376891 0.00003")
   list(APPEND DET_DIAMOND_VMCBATCH_SCALARS "eeenergy" "-3.00267948 0.000003")
   list(APPEND DET_DIAMOND_VMCBATCH_SCALARS "ionion" "-12.77567012 0.000001")
-  list(APPEND DET_DIAMOND_VMCBATCH_SCALARS "localecp" "-7.99462588 0.000007")
-  list(APPEND DET_DIAMOND_VMCBATCH_SCALARS "nonlocalecp" "0.70920650 0.000003")
+  list(APPEND DET_DIAMOND_VMCBATCH_SCALARS "localecp" "-7.99462588 0.00003")
+  list(APPEND DET_DIAMOND_VMCBATCH_SCALARS "nonlocalecp" "0.70920650 0.00001")
```

Comment:
In the failed result checking of the last run, I saw all the values with 2X of the error bars before your changes. Can we have more conservative choice of 3X? Or you have a particular reason having up-to 6X?

Reply:
I just increase as much as the deviation from the reference for the safety. I can adjust the error to fit the tolerance not the deviation.